src/library_pthread.js

```diff
@@ -242,6 +242,8 @@ var LibraryPThread = {
             Module['print']('Thread ' + e.data.threadId + ': ' + e.data.text);
           } else if (e.data.cmd === 'printErr') {
             Module['printErr']('Thread ' + e.data.threadId + ': ' + e.data.text);
+          } else if (e.data.cmd == 'alert') {
```

Comment:
I think it might be a little nicer to just call `alert()` here directly, as opposed to on Module. The reason is that `print` and `printErr` are not core web functionality, that is, how they work must be defined by the HTML. But alert is already part of the web API.